Sources/WordPressUI/Extensions/Gravatar/Gravatar.swift

```diff
@@ -30,6 +59,35 @@ public struct Gravatar {
 
         return true
     }
+    
+    /// Returns the Gravatar URL, for a given email, with the specified size + rating.
+    ///
+    /// - Parameters:
+    ///     - email: the user's email
+    ///     - size: required download size
+    ///     - rating: image rating filtering
+    ///
+    /// - Returns: Gravatar's URL
+    ///
+    public static func gravatarUrl(for email: String, size: Int? = nil, rating: GravatarRatings = .default) -> URL? {
```

Comment:
We should probably also include an option to set the placeholder (see https://en.gravatar.com/site/implement/images/). I know that some parts of the app (mention suggestions) use `identicon`.

Reply:
Done! I didn't add all the default image options, but left a link that shows all available options

https://github.com/wordpress-mobile/WordPressUI-iOS/pull/133/commits/a1829722cf3ef60035123e743c577944ac60a682